src/plugins/kibana_react/public/validated_range/validated_dual_range.tsx

```diff
@@ -35,8 +35,8 @@ interface Props extends Omit<EuiDualRangeProps, 'value' | 'onChange' | 'min' | '
   label?: string;
   formRowDisplay?: EuiFormRowDisplayKeys;
   onChange?: (val: [string, string]) => void;
-  min?: ValueMember;
-  max?: ValueMember;
+  min?: number;
+  max?: number;
```

Comment:
Not sure, and I'm not sure that I want to dig into in this PR. Was only touching this file to make tests pass... 

Seems like it might trace back to an EUI type issue where it's flexible about what types it accepts in some areas but not in others creating the awkward sometimes `string | number` other times `number` type checking...